### ISSUE TITLE
Character suggestion: ✠, 'Maltese Cross' (U+2720)

### ISSUE THREAD

**Issue by adunning** [psb1558/Junicode-font#148]
A line-height cross character, seemingly best transcribed with Unicode as ✠ ('Maltese Cross', U+2720) is used in liturgical books; as a *signe de renvoie*; and in seals to denote the beginning of a legend or as a word separator:

![Bodl. MS. Lat. liturg. e. 13, fol. 15r](https://user-images.githubusercontent.com/1084407/189698645-d6398892-7ef0-48e5-8488-5e045ac1122f.png)

[Bodl. MS. Lat. liturg. e. 13, fol. 15r](https://digital.bodleian.ox.ac.uk/objects/634a19d0-037e-4c9b-bb88-d80bbf796283/surfaces/80559c41-61e7-4987-b90a-e742f6de5d3d/)

![Bodl. MS. Junius 1, fol. 12v](https://user-images.githubusercontent.com/1084407/189696904-6736454d-14dd-4416-8b7a-ea20a212fa6e.png)

[Bodl. MS. Junius 1, fol. 12v](https://digital.bodleian.ox.ac.uk/objects/90a06f70-880a-4b5b-bd30-798710afff11/surfaces/7a3b5adb-3192-425e-ad20-4238235fbce1/)

![frideswide-seal](https://user-images.githubusercontent.com/1084407/189688008-bcbdb87e-86fe-48e2-9fd6-119559e5192b.png)

Would you please consider including this in Junicode, and possibly even Elstob if you think it appropriate? The character can be found in the original Fell Types:

![Fell Types: Great Primer Roman](https://user-images.githubusercontent.com/1084407/189694056-01260fa0-08b9-4fd4-8a18-b83f50c1a3f2.jpeg)

This is from Stanley Morison, *John Fell, The University Press and the Fell Types* (Oxford, 1967), p. 154. IM Fell has a version in its pica font under U+E01C.

Junicode has ᛭ ('Runic Cross Punctuation', U+16ED), but it is too small for these purposes, as you can see from the specimen.

As always, many thanks for your splendid work on this project!

**Comment by psb1558** [comment]
Sounds like a good idea. I like the one in the Fell sample. It seems to be about cap height:
<img width="412" alt="image" src="https://user-images.githubusercontent.com/16215767/190298753-036ab077-68e7-44c0-8f37-11af31429830.png">
As it needs to be squarish, there will be no condensed or expanded versions, and the italic and roman versions will be the same. However, there will be versions to harmonized with bold and light:
<img width="274" alt="image" src="https://user-images.githubusercontent.com/16215767/190299114-f558a7c6-c1c8-48cd-aac2-11071640212d.png">

**Comment by adunning** [comment]
This is absolutely fabulous; many thanks for your work on this! It strikes me a sensitive adaptation of the Fell version without being pedantic.

**Comment by cuddawudda** [comment]
FYI (and I know this is slightly pedantic...) the cross in the images is a "Cross pattée", not a Maltese cross. 
(A Maltese cross is 4 arrowheads)

**Comment by adunning** [comment]
Yes, 'Maltese cross' is merely the Unicode name. There is a proposal to straighten this out:

https://unicode.org/wg2/docs/n5037r-cross-patty.pdf
